### ISSUE TITLE
Double clicking on outline entry sometimes results in jumping to another item

### ISSUE THREAD

**Issue by ulugbekna** [microsoft/vscode#206424]
See in the screencast how I double click on `_extensionUri`, cursor jumps to that item's `range.start`, but because of "follow cursor", the outline entry focus changes to `constructor`

(Testing with typescript extension-provided outline)

https://github.com/microsoft/vscode/assets/16353531/8394e832-ae09-4164-84be-d659ef96e0b5

(assigning myself as well in case I get around to fixing this)

Version: 1.87.0-insider
Commit: 41e601871ac9e197b6a9245072994f6904addb60
Date: 2024-02-27T10:45:58.586Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Darwin arm64 23.3.0

**Comment by jrieken** [comment]
@joaomoreno This might be an list/tree issue (as well). I can see that the `open` event is fired twice when double clicking. This [check for `isDoubleClick`](https://github.com/microsoft/vscode/blob/7a8f2191222169dfa6cfd871a135f006a42166d9/src/vs/platform/list/browser/listService.ts#L740) does seem to be working.

**Comment by joaomoreno** [comment]
Yeah maybe `onMouseDblClick` needs to check for `this.openOnSingleClick` as well. 🙈
